src/index.js

```diff
@@ -31,15 +33,27 @@ function installFiles(sourceDir, done) {
   // The path to the package running the 'install' or 'postinstall' script.
   var fileInstallingPackagePath = hostPackageDir(scriptPath);
 
-  // The path to the package into which we should install the files.
-  var destinationDir = fileInstallingPackagePath && hostPackageDir(fileInstallingPackagePath);
-  if (!destinationDir) {
+  var source, target;
+  switch (npmv.majorVersion()) {
+    case '3':
```

Comment:
Let's actually invert the cases (`default -> '2'`, `'3' -> default`) since presumably npm will keep their format v3 format going forward.

It'd probably be good to make the new `'2'` branch fall through to a `'1'` branch too although I actually don't know what the v1 format was or even if anyone uses v1 anymore haha.